gnucash/gnome/gnc-plugin-page-budget.c

```diff
@@ -172,11 +178,17 @@ static GtkActionEntry gnc_plugin_page_budget_actions [] =
         G_CALLBACK(gnc_plugin_page_budget_cmd_allperiods_budget)
     },
     {
-        "BudgetNoteAction", "system-run", N_("Edit Note"),
+        "BudgetNoteAction", "text-x-generic", N_("Edit Note"),
         NULL,
         N_("Edit note for the selected account and period"),
         G_CALLBACK (gnc_plugin_page_budget_cmd_budget_note)
     },
+    {
+        "BudgetReportAction", "system-run", N_("Budget Report"),
+        NULL,
+        N_("Run budget report"),
```

Comment:
Translation toolkit will complain a missing full stop and a user might want an answer to "what will happen, if I press the button a second time?"